ArduCopter/mode.cpp

```diff
@@ -425,6 +425,22 @@ void Copter::update_flight_mode()
 #endif
 
     flightmode->run();
+
+#if QUICKTUNE_ENABLED
+    if (rc().find_channel_for_option(RC_Channel::AUX_FUNC::QUICKTUNE) != nullptr){
+        //Only run if the aux function is set
+        if (quicktune == nullptr && !arming.armed){
+            GCS_SEND_TEXT(MAV_SEVERITY_INFO, "Initialising Quicktune.");
+            quicktune = NEW_NOTHROW AP_Quicktune();
+            if (quicktune == nullptr) {
+                AP_BoardConfig::allocation_error("Quicktune");
+            }
+            AP_Param::load_object_from_eeprom(quicktune, quicktune->var_info);
```

Comment:
as discussed on the call, we need to trigger this in RC_Channel_Copter, as a edge triggered change

Reply:
Done.